src/HydraClient.ts

```diff
@@ -0,0 +1,124 @@
+import {hydra} from "./namespaces";
+import {IMetadataProvider} from "./DataModel/IMetadataProvider";
+import {IApiDocumentation} from "./DataModel/IApiDocumentation";
+import {IMetadataDescription} from "./DataModel/IMetadataDescription";
+import ApiDocumentation from "./ApiDocumentation";
+
+/**
+ * @class HydraClient Represents a Hypermedia Driven Application (HyDrA) Core Vocabulary client.
```

Comment:
Let's just use "Hydra" here without spelling it out.  We could add a link back to our homepage though